spec/models/user_spec.rb

```diff
@@ -87,13 +89,53 @@
   end
 
   describe 'methods' do
+    describe '#attended_event?' do
+      context 'user has attended to the event' do
+        before { events_registration.update_attributes(attended: true) }
```

Comment:
I know, but if you look at RSpec documentation it is written the other way around: https://www.relishapp.com/rspec/rspec-core/v/2-2/docs/hooks/before-and-after-hooks So when someone read the code you wrote he will probably go to the documentation, so following it would make much easier for him to understand your code.


And you also find places where it is done as I said: https://github.com/amiedes/osem/blob/83f132f84c4063091283066afc5bc00353006237/spec/models/user_spec.rb#L262